src/ui/public/courier/courier.js

```diff
@@ -114,10 +115,9 @@ uiModules.get('kibana/courier')
         }
       };
 
-      // Listen for refreshInterval changes
-      $rootScope.$watchCollection('timefilter.refreshInterval', function () {
-        const refreshValue = _.get($rootScope, 'timefilter.refreshInterval.value');
-        const refreshPause = _.get($rootScope, 'timefilter.refreshInterval.pause');
+      timefilter.on('refreshIntervalUpdate', function () {
```

Comment:
Can we use `$rootScope.$listen(timefilter, 'refreshIntervalUpdate', function () {` here so the listener is cleaned up with the `$rootScope`, which happens between each test in the browser tests.